.gitignore

```diff
@@ -2,3 +2,4 @@
 /config.local.php
 /tmp
 /logs
+/.idea/*
```

Comment:
you should put this into your [global gitignore](https://gist.github.com/cebe/1166792#file-gitconfig-L27) file. The repo should only contain repo specific ignores.